staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -125,6 +125,12 @@ type Event struct {
 	RequestReceivedTimestamp metav1.MicroTime
 	// Time the request reached current audit stage.
 	StageTimestamp metav1.MicroTime
+
+	// Annotations is an unstructured key value map stored with an event. Mechanisms like RBAC and
+	// PSP will store admission information in Annotations, like name of the policy which admits
```

Comment:
I suggest the following text instead:

> Annotations is an unstructured key value map stored with an audit event that may be set by plugins invoked in the request serving chain, including authentication, authorization and admission plugins. Keys should uniquely identify the informing component to avoid name collisions (e.g. podsecuritypolicy.admission.k8s.io/policy). Values should be short. Annotations are included in the Metadata level.

Reply:
Done.